src/SFML/Window/WindowBase.cpp

```diff
@@ -75,6 +76,37 @@ WindowBase::~WindowBase()
 }
 
 
+////////////////////////////////////////////////////////////
+WindowBase::WindowBase(WindowBase&& windowBase) noexcept :
+m_impl(std::exchange(windowBase.m_impl, nullptr)),
+m_size(std::exchange(windowBase.m_size, {}))
+{
+    // Check fullscreen style and set to new object
+    if (&windowBase == getFullscreenWindow())
+        setFullscreenWindow(this);
+
+    windowBase.close();
```

Comment:
In the case of a derived class with additional data, calling this virtual method might release resources before having a chance to move them to the new instance. I confirmed with a debugger this actually happens with `m_context` in `sf::Window` class. I suggest not calling this method.
```suggestion
```